### ISSUE TITLE
Dependency error while using environment_master_gpu.yml

### ISSUE THREAD

**Issue by krishvishal** [Calamari-OCR/calamari#48]
I've clone the repo and executed the following command.
`conda env create -f environment_master_gpu.yml`

I've encountered the following error.

```bash
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - mkl==2019.0=118
  - numpy==1.15.2=py36h1d66e8a_1 -> numpy-base==1.15.2=py36h81de0dd_1 -> mkl[version='>=2018.0.3,<2019.0a0']
Use "conda info <package>" to see the dependencies for each package.

```
I think this can be fixed by downgrading mkl version. But I want to bring this into the view of developers to get an official fix.

**Comment by ChWick** [comment]
Related to #46

**Comment by MrOCR** [comment]
@krishvishal yeah, I will update soon Stay tuned. I was busy.
krish `Star` & `Watch` this Project to get updates in the future.

**Comment by MrOCR** [comment]
@ChWick close this topic

**Comment by krishvishal** [comment]
@MrOCR will the installation work now ?

**Comment by ChWick** [comment]
@krishvishal I just merged the pull request #50 into the master branch. The installation should work now.
